env-defaults

```diff
@@ -147,6 +147,12 @@ WORK_REAPER_PERIOD_SEC=360
 # WorkItems and WorkflowSteps (in a terminal state) older than this many minutes are checked by the work reaper
 REAPABLE_WORK_AGE_MINUTES=1440
 
+# The time (in seconds) between invocations of the work failer service
+WORK_FAILER_PERIOD_SEC=360
+
+# WorkItems and associated job (in a running state) older than this many minutes are set to failed by the work failer
+FAILABLE_WORK_AGE_MINUTES=20
```

Comment:
I would make this a giant number by default so we don't accidentally fail legitimate but slow requests - I think we allowed steps in Argo workflows 4 hours by default, so maybe do the same here and use 240 minutes.